akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -211,20 +211,41 @@ akka.cluster.auto-down-unreachable-after = 120s
 This means that the cluster leader member will change the `unreachable` node
 status to `down` automatically after the configured time of unreachability.
 
-This is a naïve approach to remove unreachable nodes from the cluster membership. It
-works great for crashes and short transient network partitions, but not for long network
-partitions. Both sides of the network partition will see the other side as unreachable
```

Comment:
I think I'd like to keep "It works for for crashes and short transient network partitions, but not for long network partitions."

Reply:
My concern is that people will read that and believe that long network partitions don't apply to them, or that they are okay with manual intervention when they happen.